tests/full/test_storytelling.py

```diff
@@ -0,0 +1,334 @@
+'''A temporary test class for Daniel's storytelling project'''
+
+import nose.tools
+import tests.utilities
+
+TEST_STORY_ID_1 = 'st_1'
```

Comment:
I found this manual enumeration of test ids and documents to be a chore, which is why I put simple utility functions in place.  You probably don't want them on the utility object, but you could imagine making a local function that generated these from an index, rather than listing them out.

Your call, I'll merge this.